BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -0,0 +1,1041 @@
+                    {
+                        currentStatus = "Maneuvering (Kill Velocity)";
+                        while (UnderTimeLimit() && targetVessel != null && !complete)
+                        {
+                            relVel = targetVessel.GetObtVelocity() - vessel.GetObtVelocity();
+                            fc.attitude = (relVel + targetVessel.acceleration).normalized;
+                            complete = relVel.magnitude < firingSpeed / 3;
+                            fc.throttle = !complete ? 1 : 0;
+
+                            yield return new WaitForFixedUpdate();
+                        }
+                    }
+                    else if (hasPropulsion && targetVessel != null && AngularVelocity(vessel, targetVessel) > firingAngularVelocityLimit)// && currentProjectile != null
+                    {
+                        currentStatus = "Maneuvering (Kill Angular Velocity)";
+
+                        while (UnderTimeLimit() && targetVessel != null && !complete)
+                        {
+                            complete = AngularVelocity(vessel, targetVessel) < firingAngularVelocityLimit / 2;
+                            fc.attitude = Vector3.ProjectOnPlane(RelVel(vessel, targetVessel), FromTo(vessel, targetVessel)).normalized * -1;
+                            fc.throttle = !complete ? 1 : 0;
+
+                            yield return new WaitForFixedUpdate();
+                        }
+                    }
+                    else
+                    {
+                        if (hasPropulsion)
+                        {
+                            if (currentRange < minRange)
+                            {
+                                currentStatus = "Maneuvering (Drift Away)";
+
+                                Vector3 toTarget;
+                                fc.throttle = 0;
+
+                                while (UnderTimeLimit() && targetVessel != null && !complete)
+                                {
+                                    toTarget = FromTo(vessel, targetVessel);
+                                    complete = toTarget.magnitude > minRange;
+                                    fc.attitude = toTarget.normalized;
+
+                                    yield return new WaitForFixedUpdate();
+                                }
+                            }
+                            else
+                            {
+                                currentStatus = "Maneuvering (Drift)";
+                                fc.throttle = 0;
+                                fc.attitude = Vector3.zero;
+                            }
+                        }
+                        else
+                        {
+                            currentStatus = "Stranded";
+                            fc.throttle = 0;
+                            fc.attitude = Vector3.zero;
+                        }
+
+                        yield return new WaitForSeconds(updateInterval);
```

Comment:
These and any others that should run in `FixedUpdate` should be switched to use `WaitForSecondsFixed`.